### PR TITLE
updated README.md

### LINKED ISSUE #263: Readme.md file can be more fascinating
### Description

Readme.md file can be more fascinating by adding contributors section, Tech stack section and a working video of our project 
@krishnaacharyaa sir assign this issue to me i will add it 

### Current Behavior

missing sections

### Proposed Enhancement

Will add some sections

### Use Case

Will look good

### Blockers

_No response_

### Additional Information/ Attachments

_No response_

### PR DESCRIPTION
**Pull request by Meetjain1**
## Summary
updated readme.md

## Issue(s) Addressed
closes #263

### CODE REVIEW

**Review by krishnaacharyaa — CHANGES_REQUESTED** [review]
All left aligned please.
And make sure every emojis are bouncing and animated if possible like the open source wala muscle
And instead of :- make it : 

There are other content changes, i'll let you know after some time may be tomorrow @Meetjain1

**Comment by Meetjain1** [comment]
> All left aligned please. And make sure every emojis are bouncing and animated if possible like the open source wala muscle And instead of :- make it :
> 
> There are other content changes, i'll let you know after some time may be tomorrow @Meetjain1

sure

**Comment by Meetjain1** [comment]
> All left aligned please. And make sure every emojis are bouncing and animated if possible like the open source wala muscle And instead of :- make it :
> 
> There are other content changes, i'll let you know after some time may be tomorrow @Meetjain1

done with these  changes ,let me know about the other changes @krishnaacharyaa

**Comment by krishnaacharyaa** [comment]
image

Can we add contributing guidelines even here?

File: README.md [hunk 1]
```diff
@@ -1,106 +1,122 @@
...
 ![Preview Image](https://github.com/krishnaacharyaa/wanderlust/assets/116620586/17ba9da6-225f-481d-87c0-5d5a010a9538)
 
-## [Figma Design File](https://www.figma.com/file/zqNcWGGKBo5Q2TwwVgR6G5/WanderLust--A-Travel-Blog-App?type=design&node-id=0%3A1&mode=design&t=c4oCG8N1Fjf7pxTt-1)
-## [Discord Channel](https://discord.gg/FEKasAdCrG)
+<hr>
 
-## 🎯 Goal of this project
+<div>
+<h3>• Watch our collaboration video : </h3>
+  <a href="https://youtu.be/ANfC1u_N_A0?feature=shared"> <img src="https://github.com/krishnaacharyaa/wanderlust/assets/133582566/47d71dd6-0390-479e-9d4e-3f077ef1a987.png" width="45" height="40"></a> <h4> with TechwithShubham </h4>
```
Comment (krishnaacharyaa) [inline README.md:12]: Remove that dot and : from all the headers

File: README.md [hunk 2]
```diff
@@ -1,106 +1,122 @@
...
-## 🎯 Goal of this project
+<div>
+<h3>• Watch our collaboration video : </h3>
+  <a href="https://youtu.be/ANfC1u_N_A0?feature=shared"> <img src="https://github.com/krishnaacharyaa/wanderlust/assets/133582566/47d71dd6-0390-479e-9d4e-3f077ef1a987.png" width="45" height="40"></a> <h4> with TechwithShubham </h4> 
+  For an in-depth look! 🎬 Dive into the details and discover more about our exciting project.
+</div>
+<hr>
 
-At its core, this project embodies two important aims:
+## About :
```
Comment (krishnaacharyaa) [inline README.md:18]: Remove this section

File: README.md [hunk 3]
```diff
@@ -1,106 +1,122 @@
...
 
-1. **Start Your Open Source Journey**: It's aimed to kickstart your open-source journey. Here, you'll learn the basics of Git and get a solid grip on the MERN stack and I strongly believe that learning and building should go hand in hand.
-2. **React Mastery**: Once you've got the basics down, a whole new adventure begins of mastering React. This project covers everything, from simple form validation to advanced performance enhancements. And I've planned much more cool stuff to add in the near future if the project hits more number of contributors.
+**• WanderLust** is a straightforward MERN travel blog website ✈. This project aims to help people contribute to open source, enhance their  React skills, and master Git. 
+<hr>
 
-_I'd love for you to make the most of this project - it's all about learning, helping, and growing in the open-source world._
+<div>
+<h3>• Figma Links of Our Project :</h3>
+  <a href="https://www.figma.com/file/zqNcWGGKBo5Q2TwwVgR6G5/WanderLust--A-Travel-Blog-App?type=design&node-id=0%3A1&mode=design&t=c4oCG8N1Fjf7pxTt-1">
```
Comment (krishnaacharyaa) [inline README.md:24]: Remove * and then make this more interesting somehow. 

We need a section like important links which will have yotuube, discord and figma link. Soon after the first image. this should be there.

You can come up with innovation.
I see it something like table with 2 cols, 1st column logo and 2nd column description and link
Discord Logo | Join our project's discord link [here]()
Figma Logo | ---
youtube logo | --

Think about it it should be attractive at the same time appealing

File: README.md [hunk 4]
```diff
@@ -1,106 +1,122 @@
...
-   ```
+• Feel free to get in touch with me for any further queries or support, happy to help :)
 
-4. **Launch the Development Server**
+<hr>
 
-   ```bash
-   npm run dev
-   ```
+## 📞Contact
```
Comment (krishnaacharyaa) [inline README.md:102]: space between the things, but we can remove this completely my x and linkendin not required here

**Review by krishnaacharyaa — CHANGES_REQUESTED** [review]
LGTM, some minor changes

**Comment by krishnaacharyaa** [comment]
And discord link is invalid

File: README.md (hunk 2 above)
Comment (Meetjain1) [reply README.md:18]: About section?

File: README.md (hunk 1 above)
Comment (Meetjain1) [reply README.md:12]: okk done

File: README.md (hunk 2 above)
Comment (Meetjain1) [reply README.md:18]: okk done

File: README.md (hunk 3 above)
Comment (Meetjain1) [reply README.md:24]: okk done

**Comment by Meetjain1** [comment]
@krishnaacharyaa have a look now , I think I have completed all the changes you want .

**Comment by Meetjain1** [comment]
> image
> 
> Can we add contributing guidelines even here?

I dont think that we can do this by editing readme.md file

**Comment by Meetjain1** [comment]
> Why is there some space in the right side not used in the contributions section? can we make it a little small, only a little @Meetjain1

okk done

**Comment by Meetjain1** [comment]
> Looks very nice, GSSoC label , i am thinking can we make it little revealing? may be adding it in the important links? what do you feel and how about having 🔗 than gem in the links emoji section @Meetjain1

adding that jam of link , but gssoc label important links mai hmm.. ok link kaunsi du gssoc mai ? gssoc 24 ki ?

**Comment by krishnaacharyaa** [comment]
> > Looks very nice, GSSoC label , i am thinking can we make it little revealing? may be adding it in the important links? what do you feel and how about having 🔗 than gem in the links emoji section @Meetjain1
> 
> adding that jam of link , but gssoc label important links mai hmm.. ok link kaunsi du gssoc mai ? gssoc 24 ki ?

No we'll have to figure out, the gssoc has deployed our project in their some website, that link we'll have to put or else if you feel link mei doesn't look good.

Convert the existing description like table of 2 columns , 
GSSOC logo | description 

logo should span across 2 rows got it? @Meetjain1

**Comment by Meetjain1** [comment]
@krishnaacharyaa have a look now , I think no changes required now 🙃

**Comment by krishnaacharyaa** [comment]
@Meetjain1 , the link wala why did you change? I am not able to click anything? 

The previous one the content was all good, the blue underline and proper links and stuff, 

I wanted you to just change the heading from 💎 to 🔗. Rest all looks really good.
Once this is done. kindly raise fresh PR which follows contributing guidelines, so that i can close this as soon as possible. so that you get your points.

**Comment by Meetjain1** [comment]
> @Meetjain1 , the link wala why did you change? I am not able to click anything?
> 
> The previous one the content was all good, the blue underline and proper links and stuff,
> 
> I wanted you to just change the heading from 💎 to 🔗. Rest all looks really good. Once this is done. kindly raise fresh PR which follows contributing guidelines, so that i can close this as soon as possible. so that you get your points.

arae mujhe laga aap ye change krne ko bole thae

**Comment by krishnaacharyaa** [comment]
The previous one was good bro! Blue underline just heading Kindly change

Once you raise new PR which follows the contributing guidelines I'll merge this 

Even I'm eager to get this changes in 😍😊

No more changes hopefully @Meetjain1

**Comment by Meetjain1** [comment]
> The previous one was good bro! Blue underline just heading Kindly change
> 
> Once you raise new PR which follows the contributing guidelines I'll merge this
> 
> Even I'm eager to get this changes in 😍😊
> 
> No more changes hopefully @Meetjain1

Haash ok 👍

**Comment by Meetjain1** [comment]
@krishnaacharyaa pr title checker ka issue aaraha hai kya hai ye ?

**Comment by krishnaacharyaa** [comment]
@Meetjain1 slow down your horses, 5th time I'm saying take your time and raise new PR reading the 5 points in contributing guidelines.

Once your PR follows the commits and PR title according to guidelines will merge this.

**Comment by krishnaacharyaa** [comment]
@Meetjain1 pardon me, missed the new PR!. Closing this

**Pull request closed without merging**